linkerd/timeout/src/failfast.rs

```diff
@@ -128,8 +133,8 @@ where
             ret => {
                 match self.state {
                     State::Open => {}
-                    State::Waiting => trace!("Ready"),
-                    State::FailFast => debug!("Recovered"),
+                    State::Waiting => trace!("{} has become ready", self.scope),
+                    State::FailFast => debug!("{} service has recovered", self.scope),
```

Comment:
should we consider bumping this one up to info? since folks are going to actually see the "entering failfast" message in their logs, it might be worth also letting them know when the service _leaves_ failfast, so they don't assumeit has failed forever?